src/test/java/reposense/authorship/AnnotatorAnalyzerTest.java

```diff
@@ -189,6 +232,10 @@ public void extractAuthorName_matchCommentPattern4_returnAuthorName() {
         line = "  %  @@author   fake-4u-th0r  ";
```

Comment:
You may want to include a test case for ` line = " % @@author fake-4u-th0r"` (without the ending whitespace) in order to check the new comment format of
```
{"%", null}
```

Reply:
In addition, please update https://reposense.org/ug/usingAuthorTags.html to reflect the use of Git author names rather than GitHub IDs.

Reply:
Hi, sorry for the late response, but can I check if this case is covered by line 224 (`line = "%@@author fakeAuthor";`)? Because the case without the ending whitespace I think is sufficiently covered by that.

`line = " % @@author fake-4u-th0r"` Or did you mean to have a case where there is a whitespace at the beginning, and no whitespace at the end?

Reply:
Will update the necessary md documents, thanks!

Reply:
> Or did you mean to have a case where there is a whitespace at the beginning, and no whitespace at the end?

Yup.